pkgs/applications/version-management/git-and-tools/gh/default.nix

```diff
@@ -27,6 +27,16 @@ buildGoModule rec {
     done
   '';
 
+  # should be in the next release
+  # https://github.com/cli/cli/issues/401#issuecomment-586918622
+  # https://github.com/cli/cli/issues/401#issuecomment-588261787
```

Comment:
copy pasta error
```suggestion
```

Reply:
It isn't an error, linking to two separate comments in the thread by the upstream maintainer.

Reply:
oh, the comments are different, didn't notice